relay-dynamic-config/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "relay-dynamic-config"
+authors = ["Sentry <oss@sentry.io>"]
+description = "Dynamic configuration passed down from sentry"
+homepage = "https://getsentry.github.io/relay/"
+repository = "https://github.com/getsentry/relay"
+version = "23.1.1"
+edition = "2021"
+license-file = "../LICENSE"
+publish = false
+
+[features]
+default = []
+processing = []
+
+[dependencies]
+# TODO: Move types used by ProjectConfig from their respective crates to this one. Eventually,
+# this crate should not depend on any sibling crates, except maybe relay-common.
+relay-auth = { path = "../relay-auth" }
+relay-common = { path = "../relay-common" }
+relay-filter = { path = "../relay-filter" }
+relay-general = { path = "../relay-general" }
+relay-sampling = { path = "../relay-sampling" }
```

Comment:
These are the crates that `relay-cabi` also depends on, so having them here should be acceptable for now.